src/util/authUtils.test.js

```diff
@@ -1,4 +1,5 @@
 import Joi from 'joi';
+import config from './config';
```

Comment:
probably not great to have unit tests depend on environment config - we should always mock the config in tests